pkg/utils/utils.go

```diff
@@ -237,6 +238,44 @@ func MergeStringMap(dest, src map[string]string) {
 	}
 }
 
+const (
+	errUnexpectedKey = "unexpected key in data: %s"
+	errSecretType    = "can not handle secret value with type %T"
```

Comment:
Since this is now an extracted method, I think the Errors should also be extracted if someone wants to check against them outside this package? WDYT?